packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import Cookies from 'js-cookie'
+
+export abstract class BaseStorage {
+    abstract set(key: string, value: string, options?: any): void
+    abstract get(key: string): string
+    abstract delete(key: string): void
+}
+
+export class CookieStorage extends BaseStorage {
+    constructor() {
+        super()
+        if (typeof document === 'undefined') {
+            throw new Error('CookieStorage is not avaliable on the current environment.')
+        }
+    }
+    set(key: string, value: string, options?: Cookies.CookieAttributes) {
+        Cookies.set(key, value, {secure: true, ...options})
+    }
+    get(key: string) {
+        return Cookies.get(key) || ''
+    }
+    delete(key: string) {
+        Cookies.remove(key)
+    }
+}
+
+export class LocalStorage extends BaseStorage {
+    constructor() {
+        super()
+        if (typeof window === 'undefined') {
+            throw new Error('LocalStorage is not avaliable on the current environment.')
+        }
+    }
+    set(key: string, value: string, options?: any) {
```

Comment:
```suggestion
    set(key: string, value: string) {
```
It's not used, so I don't think you need to declare it.

Reply:
✅